modules/disabling-crd-preflight.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * extensions/ce/crd-upgrade-safety.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="disabling-crd-preflight_{context}"]
+= Disabling CRD upgrade safety preflight check
+
+The custom resource definition (CRD) upgrade safety preflight check can be disabled by adding the `preflight.crdUpgradeSafety.disabled` field with a value of `true` to the `ClusterExtension` object that provides the CRD.
+
+[WARNING]
+====
+Disabling the CRD upgrade safety preflight check could break backwards compatibility with stored versions of the CRD and cause other unintended consequences on the cluster.
+====
+
+You cannot disable individual field validators. If you disable the CRD upgrade safety preflight check, all field validators are disabled.
+
+[NOTE]
+====
+The following checks are handled by the Kubernetes API server:
+
+* The scope changes from `Cluster` to `Namespace` or from `Namespace` to `Cluster`
+* An existing stored version of the CRD is removed
+
+After disabling the CRD upgrade safety preflight check via {olmv1-first}, these two operations are still prevented by Kubernetes.
+====
+
+.Prerequisites
+
+* You have a cluster extension installed.
+
+.Procedure
+
+. Edit the `ClusterExtension` object of the CRD:
++
+[source,terminal]
+----
+$ oc edit clusterextension <clusterextension_name>
+----
+
+. Set the `disabled` field to `true`:
```

Comment:
nit: would using the fully qualified path for the field be more reader friendly?